src/js/app.js

```diff
@@ -0,0 +1,113 @@
+const $todoInput = document.querySelector("#new-todo-title");
+const $toggleInput = document.querySelector(`#todo-list`);
+const $todoCount = document.querySelector(`span strong`);
+
+$todoInput.addEventListener("keyup", addTodoItem);
+
+document.addEventListener("click", activeTodoItem);
+document.addEventListener("click", allTodoItem);
+document.addEventListener("click", finishTodoItem);
+
+function renderTodoItemTemplate(title) {
+    return ` <li class="active">
+                  <div class="view">
+                      <input class="toggle" onclick="onToggleTodoItem(event)" type="checkbox" >
+                      <label class="label" ondblclick="updateTodoItem(event)">${title}</label>
+                      <button class="destroy" onclick="deleteTodoItem(event)"></button>
+                  </div>
+                  <input class="edit" value=${title} />
+              </li>`;
+}
+
+function addTodoItem(event) {
+    const todoTitle = event.target.value;
+    const todoList = document.getElementById("todo-list");
```

Comment:
```suggestion
    const $todoList = document.getElementById("todo-list");
```

HTMLElement 의 변수명에는 $를 붙이는게 암묵적인(?) 컨벤션인데, 이왕 적용하시기로 하셨다면,
$todoInput, $toggleInput 과 같이 일관성있게 다 붙이시는게 좋을 것 같아요!